src/node_buffer.cc

```diff
@@ -197,7 +298,7 @@ Local<Object> New(Environment* env,
 }
 
 
-Local<Object> Use(Isolate* isolate, char* data, uint32_t length) {
+Local<Object> Use(Isolate* isolate, char* data, size_t length) {
```

Comment:
Is ABI compatibility a concern?  Because changing the type from uint32_t to size_t changes the name mangling.